src/runtime_src/core/edge/drm/zocl/cu_scu.c

```diff
@@ -34,6 +34,7 @@ struct xrt_cu_scu {
 	int			 run_cnts;
 	void			*vaddr;
 	int			 num_reg;
+	bool			 sk_crashed;
```

Comment:
Please add a commentary in Linux style on what this flag is and how is it used. What happens after a crash and is this ever reset, etc.?